### PR TITLE
Edited Version

### PR DESCRIPTION
**Pull request by HoltkampN**
Make sure unnecessary namespace usage is removed, change explicit variable to implicit var, revise loop, make new class for simplicity

### CODE REVIEW

File: RefactorKata/RefactorKata/Program.cs [hunk 1]
```diff
@@ -1,41 +1,40 @@
...
     class Program
-    {
-        static void Main(string[] args)
+    {   //remove args//
+        static void Main (string[] args) 
         {
-            //This is intentionally bad : (  Let's Refactor!
-            System.Data.SqlClient.SqlConnection Conn = new System.Data.SqlClient.SqlConnection("Server=.;Database=myDataBase;User Id=myUsername;Password = myPassword;");
+            //remove redundant using statement, unnecessary to repeat namespace, explicit to implicit "var" || "conn" = Camel Case//
+            var conn = new SqlConnection("Server=.;Database=myDataBase;User Id=myUsername;Password = myPassword;");
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: I've like to see this connection string removed and utilize ConfigurationManager.ConnectionStrings

Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: This needs the "using" keyword

File: RefactorKata/RefactorKata/Program.cs [hunk 2]
```diff
@@ -1,41 +1,40 @@
...
 
-            System.Data.SqlClient.SqlCommand cmd = Conn.CreateCommand();
+            var cmd = conn.CreateCommand();
             cmd.CommandText = "select * from Products";
-            /*
+            /* Is this line necessary @NHoltkamp
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: Commented code with no explanation should be removed

File: RefactorKata/RefactorKata/Program.cs [hunk 3]
```diff
@@ -1,41 +1,40 @@
...
              * cmd.CommandText = "Select * from Invoices";
              */
-            System.Data.SqlClient.SqlDataReader reader = cmd.ExecuteReader();
-            List<Product> products = new List<Product>();
+            var reader = cmd.ExecuteReader();
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: I'd like to see this Extracted to a GetProducts method

File: RefactorKata/RefactorKata/Program.cs (hunk 1 above)
Comment (HoltkampN) [reply RefactorKata/RefactorKata/Program.cs]: I'm happy to do so, but there is no app.config, please advise.

File: RefactorKata/RefactorKata/Program.cs (hunk 2 above)
Comment (HoltkampN) [reply RefactorKata/RefactorKata/Program.cs]: removed comment

File: RefactorKata/RefactorKata/Program.cs (hunk 3 above)
Comment (HoltkampN) [reply RefactorKata/RefactorKata/Program.cs]: now in shortened form & added IEnumerable

File: RefactorKata/RefactorKata/Product.cs [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿namespace RefactorKata
+{
+    public class Product
+    {
+
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Product.cs]: unnecessary whitespace (super picky, but the heart wants what the heart wants).

File: RefactorKata/RefactorKata/Program.cs [hunk 5]
```diff
@@ -1,41 +1,47 @@
...
+                new SqlConnection("Server=.;Database=myDataBase;User Id=myUsername;Password = myPassword;"))
             {
-                Console.WriteLine(products[i].name);
+                var cmd = conn.CreateCommand();
+                cmd.CommandText = "select * from Products";
+
+                var reader = cmd.ExecuteReader();
+                var products = new List<Product>();
+
+
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: unnecessary whitespace (super picky, but the heart wants what the heart wants).

File: RefactorKata/RefactorKata/Program.cs [hunk 6]
```diff
@@ -1,41 +1,47 @@
...
+
+                while (reader.Read())
+                {
+
```
Comment (soitgoes) [inline RefactorKata/RefactorKata/Program.cs]: unnecessary whitespace (super picky, but the heart wants what the heart wants).

File: RefactorKata/RefactorKata/Product.cs (hunk 4 above)
Comment (HoltkampN) [reply RefactorKata/RefactorKata/Product.cs]: I appreciate the critical nature, it will only make all of us better programmers and learners in the end. I will update that soon.
